src/themes/teams/components/Radio/radioStyles.ts

```diff
@@ -0,0 +1,36 @@
+import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
+
+const radioStyles: IComponentPartStylesInput = {
+  root: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      display: 'inline-flex',
+      position: 'relative',
+      alignItems: 'center',
+      justifyContent: 'flex-end',
+      outline: 0,
+      minHeight: '2.5rem',
+    }
+  },
+
+  radio: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      display: 'inline-block',
+    }
+  },
+
+  label: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      alignItems: 'center',
+      cursor: 'pointer',
```

Comment:
one thing that I am missing with that is an ability to check `Radio` by clicking on label. Seems that we should do either of two
- do not introduce this style property now
- or introduce support for radio selection by clicking on its label